src/app/space/create/deployments/apps/deployment-details.component.ts

```diff
@@ -164,6 +160,16 @@ export class DeploymentDetailsComponent {
         .subscribe(this.hasPods)
     );
 
+    this.subscriptions.push(
+      this.hasPods.subscribe((hasPods: boolean): void => {
+        if (!hasPods) {
+          this.cpuChart.next(null);
```

Comment:
I've grown to not be a fan of using null as a state for some object. Is it possible to introduce an EMPTY_STATE that's null, or better yet an actual object that represents this?